test/test_core.py

```diff
@@ -7209,8 +7209,13 @@ def test_EXPORTED_RUNTIME_METHODS(self):
     'minimal_runtime': ['-sMINIMAL_RUNTIME=1']
   })
   def test_dyncall_specific(self, *args):
-    if self.get_setting('WASM_BIGINT') or self.get_setting('MEMORY64'):
-      self.skipTest('not compatible with WASM_BIGINT')
+    if self.get_setting('MEMORY64'):
+      self.skipTest('not compatible with MEMORY64')
+    if self.get_setting('WASM_BIGINT'):
+      # define DYNCALLS because this test does test calling them directly, and
+      # in WASM_BIGINT mode we do not enable them by default (since we can do
+      # more without them - we don't need to legalize)
+      args = list(args) + ['-sDYNCALLS=1', '-DWASM_BIGINT']
```

Comment:
I think you can just do `args +=` here

Reply:
That modifies the input array, however, which can mean these accumulate, depending on how the test runner works? With the parallelism and all that I'd rather not make assumptions there.

Reply:
But you are using *arg .. which mean python is creating an array based on the inputs.. and it will be fresh each time I believe.

At the very least you can do `args = args +  ['-sDYNCALLS=1', '-DWASM_BIGINT']` since that also makes a copy.

Reply:
I checked now to make sure. It turns out that `*args` is always a tuple, not a list. So we do need the full syntax here, to turn the tuple into a list.

Instead, we could avoid `*args`, by replacing
```
    '': [],
```
with
```
    '': ([],),
```
and it is received as `test_dyncall_specific(self, args)`, after which the code is simpler. Tradeoffs both ways, I don't feel strongly.

Reply:
Either way lgtm